sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -85,22 +86,19 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
             #region Snippet:DigitalTwinsSampleGetBasicDigitalTwin
 
             Response<BasicDigitalTwin> getBasicDtResponse = await client.GetDigitalTwinAsync<BasicDigitalTwin>(basicDtId);
-            if (getBasicDtResponse.GetRawResponse().Status == (int)HttpStatusCode.OK)
-            {
-                BasicDigitalTwin basicDt = getBasicDtResponse.Value;
-
-                // Must cast Component1 as a JsonElement and get its raw text in order to deserialize it as a dictionary
-                string component1RawText = ((JsonElement)basicDt.Contents["Component1"]).GetRawText();
-                IDictionary<string, object> component1 = JsonSerializer.Deserialize<IDictionary<string, object>>(component1RawText);
-
-                Console.WriteLine($"Retrieved and deserialized digital twin {basicDt.Id}:\n\t" +
-                    $"ETag: {basicDt.ETag}\n\t" +
-                    $"Prop1: {basicDt.Contents["Prop1"]}\n\t" +
-                    $"Prop2: {basicDt.Contents["Prop2"]}\n\t" +
-                    $"Component1 metadata: {component1[DigitalTwinsJsonPropertyNames.DigitalTwinMetadata]}\n\t" +
-                    $"Component1.Prop1: {component1["ComponentProp1"]}\n\t" +
-                    $"ComponentProp2: {component1["ComponentProp2"]}");
-            }
+            BasicDigitalTwin basicDt = getBasicDtResponse.Value;
+
+            // Must cast Component1 as a JsonElement and get its raw text in order to deserialize it as a dictionary
+            string component1RawText = ((JsonElement)basicDt.Contents["Component1"]).GetRawText();
+            var component1 = JsonSerializer.Deserialize<BasicDigitalTwinComponent>(component1RawText);
+
+            Console.WriteLine($"Retrieved and deserialized digital twin {basicDt.Id}:\n\t" +
+                $"ETag: {basicDt.ETag}\n\t" +
+                $"ModelId: {basicDt.Metadata.ModelId}\n\t" +
+                $"Prop1: {basicDt.Contents["Prop1"]} and last updated on {basicDt.Metadata.PropertyMetadata["Prop1"].LastUpdatedOn}\n\t" +
```

Comment:
This is the second easier change (no further deserialization or casting required), and can access `LastUpdatedOn` directly off the dictionary.